sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/FeatureFilterUtils.java

```diff
@@ -3,24 +3,33 @@
 
 package com.azure.spring.cloud.feature.management.implementation;
 
-import org.springframework.util.StringUtils;
-
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Map;
 
+import org.springframework.util.StringUtils;
+
 public class FeatureFilterUtils {
+    
+    public static void updateValueFromMapToList(Map<String, Object> parameters, String key) {
+        updateValueFromMapToList(parameters, key, false);
+    }
     /**
      * Looks at the given key in the parameters and coverts it to a list if it is currently a map.
      *
      * @param parameters map of generic objects
      * @param key key of object int the parameters map
      */
     @SuppressWarnings("unchecked")
-    public static void updateValueFromMapToList(Map<String, Object> parameters, String key) {
+    public static void updateValueFromMapToList(Map<String, Object> parameters, String key, boolean fixNull) {
```

Comment:
I don't see this new function being called? It's called from the (map, string) version, but that's the only one being called (as far as I can tell).

Reply:
Maybe this wasn't a V1 schema change, or a test isn't catching the issue. Removing for now.